app/services/zendesk/ticket_merging_service.rb

```diff
@@ -0,0 +1,145 @@
+module Zendesk
+  class TicketMergingService
+    include ZendeskServiceHelper
+
+    RETURN_STATUS_ORDER =
+      [
+        EitcZendeskInstance::RETURN_STATUS_UNSTARTED,
+        EitcZendeskInstance::RETURN_STATUS_FOREIGN_STUDENT,
+        EitcZendeskInstance::RETURN_STATUS_IN_PROGRESS,
+        EitcZendeskInstance::RETURN_STATUS_READY_FOR_QUALITY_REVIEW,
+        EitcZendeskInstance::RETURN_STATUS_READY_FOR_SIGNATURE_ESIGN,
+        EitcZendeskInstance::RETURN_STATUS_READY_FOR_SIGNATURE_PICKUP,
+        EitcZendeskInstance::RETURN_STATUS_READY_FOR_EFILE,
+        EitcZendeskInstance::RETURN_STATUS_COMPLETED_RETURNS
+      ]
+
+    INTAKE_STATUS_ORDER =
+      [
+        EitcZendeskInstance::INTAKE_STATUS_UNSTARTED,
+        EitcZendeskInstance::INTAKE_STATUS_IN_PROGRESS,
+        EitcZendeskInstance::INTAKE_STATUS_GATHERING_DOCUMENTS,
+        EitcZendeskInstance::INTAKE_STATUS_READY_FOR_REVIEW,
+        EitcZendeskInstance::INTAKE_STATUS_IN_REVIEW,
+        EitcZendeskInstance::INTAKE_STATUS_WAITING_FOR_INFO,
+        EitcZendeskInstance::INTAKE_STATUS_READY_FOR_INTAKE_INTERVIEW,
+        EitcZendeskInstance::INTAKE_STATUS_COMPLETE
+      ]
+
+    def instance
+      EitcZendeskInstance
+    end
+
+    def merge_duplicate_tickets(intake_ids)
+      ticket_ids = Intake.find(intake_ids).map(&:intake_ticket_id).compact
+      primary_ticket = find_primary_ticket(ticket_ids)
+
+      # this only happens if all tickets are closed
+      unless primary_ticket
+        Rails.logger.info("Could not identify primary ticket during duplicate merging (intake id's: #{intake_ids.join(", ")})")
```

Comment:
we weren't sure whether this was the best way to log things as they happen here. can someone weigh in?